scripts/build.sh

```diff
@@ -164,6 +164,17 @@ cortxfs_make() {
         exit 1;
     fi
 
+    if [ -f "$CORTXFS_SRC/management/REST_doc.html" ]; then
+	    echo "Deleting Swagger Documentation"
+	    rm "$CORTXFS_SRC/management/REST_doc.html"
+    fi
+
+    sudo pip3 install PyYaml
```

Comment:
Invoking `sudo` and `pip3` every time when we call `make` might have negative impact on the build process especially in environments where sudo requires network IO.
Please consider moving this into the configure step and consider using `--user` instead of `sudo`:

```bash
cortxfs_configure() {
  pip3 install PyYaml --user
  # ...
}
```

Even better idea is to add `python36-PyYAML` into the RPM spec file as a build dependency (`BuildRequires`) and install the deps after configure is over (or manually go over the list of deps and install them before executing confiugre). Either way, it is better to install an RPM package than downloading a package from pypi.